androidApp/src/main/java/com/mbta/tid/mbta_app/android/search/SearchBarOverlay.kt

```diff
@@ -154,7 +153,7 @@ fun SearchBarOverlay(
                     onExpandedChange = {},
                 ) {
                     LazyColumn(
-                        modifier = Modifier.fillMaxSize().background(colorResource(R.color.fill1)),
+                        modifier = Modifier.fillMaxSize().background(colorResource(R.color.fill2)),
```

Comment:
suggestion: This should stay fill1  based on [this design QA](https://www.figma.com/design/OVO5Z0eYUtnnRvbxsIRpO3/App-QA?node-id=1013-221&t=tPCPdpf79064Fjj7-0)